tests/ConfigurationTest.php

```diff
@@ -5,6 +5,7 @@
 use InvalidArgumentException;
 use PHPUnit\Framework\TestCase;
 use Cake\Chronos\Chronos;
+use PHP_CodeSniffer\Config;
```

Comment:
Do we need the CodeSniffer Config class?

Reply:
Sorry. It's my fault and I've removed :).

Reply:
Ok thanks!